src/cudamatrix/cu-matrix.cc

```diff
@@ -2183,6 +2335,39 @@ Real CuMatrixBase<Real>::Trace(bool check_square) const {
   }
 }
 
+template <typename Real>
+void CuMatrixBase<Real>::CopyFromGeneralMat(const GeneralMatrix &src,
+                                            MatrixTransposeType trans) {
+  switch (src.Type()) {
+    case kFullMatrix: {
+      const Matrix<BaseFloat> &src_full_mat = src.GetFullMatrix();
+      this->CopyFromMat(src_full_mat, trans);
+      return;
+    }
+    case kCompressedMatrix: {
+      Matrix<BaseFloat> mat;
+      src.GetMatrix(&mat);
+      this->CopyFromMat(mat, trans);
```

Comment:
`this->` is at the least inconsistently used, see `Mat` below without the `this->` qualifier.